internal/testrunner/coverage_output.go

```diff
@@ -0,0 +1,248 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Nit. Underscores shouldn't be used in go file names, unless they mean something for `go build`. This file could be named `coverageoutput.go`.

Reply:
I'm afraid we're overusing them here. I will adjust this file unit.

Do you know if there is any official recommendation written down? I must have missed reading it.

Reply:
I think there is not, but for packages (and then for directories), there is a recommendation to don't use underscores. And for files, some suffixes have special meanings for go build, as the ones for architectures or operating systems, so in general I think that is a good idea to avoid them.

Reply:
Thanks for the suggestion, fixed!